woocommerce/woocommerce-admin
# Setup Checklist: Update PayU logo

Fixes #6780

This pull request replaces the existing PayU logo in the setup checklist with the new one supplied in the issue linked above.

__Before:__
<img width="348" alt="pay-u-before" src="https://user-images.githubusercontent.com/22080/115302164-fbc0c880-a116-11eb-9974-4ad708200991.png">

__After:__
<img width="348" alt="pay-u-after" src="https://user-images.githubusercontent.com/22080/115302190-011e1300-a117-11eb-9a0d-4c33bfe50073.png">

### Detailed test instructions:

-   Launch the OBW, and enter a store address in India
-   Visit the home screen, with the setup checklist displayed and enter the payments task. Verify the new image seen above is shown.
